scripts/pages.js

```diff
@@ -0,0 +1,130 @@
+const moment = require('moment');
+const Routes = require('../src/js/containers/router/RouterRoutes.js').routes;
+
+const recipientRequestObject = {
+    name: 'recipient',
+    updatedFrequency: 'daily',
+    priority: '0.8',
+    isAsync: true,
+    url: 'https://api.usaspending.gov/api/v2/recipient/duns/',
```

Comment:
Since we're coupled to the prod API here, and coupled further to specific web request syntax for a few API endpoints, there is potential for breakage here. 

If it breaks, it may stay broken for a while before we realize it (especially if this does not block build or deploy). 

Are there any tests of sitemap generation that can ensure the way we have it coded _will/should_ still work when it comes time to generate it for the production deploy? That way we'd realize it at unittest-time.

Reply:
Not currrently.